packages/test-generator/integration-test-templates/cypress/integration/generate-spec.ts

```diff
@@ -142,7 +142,12 @@ const EXPECTED_SUCCESSFUL_CASES = new Set([
   'DataStoreActions',
   'FormWithState',
   'InternalMutation',
-  'TwoWayBindings',
+  'FullForm',
+  'Listings',
+  'ListingComp',
+  'RoudedImage',
```

Comment:
Probably don't need/want to commit this.

Reply:
Messed up the rebase. Will fix